artik10.coffee

```diff
@@ -15,6 +15,7 @@ postProvisioningInstructions = [
 ]
 
 module.exports =
+	version: 'v1'
```

Comment:
I think we should omit the `v` in this property (for consistency with how other projects do versioning, and you can plug them directly to version comparators modules without any previous cleanup). Are we going to follow semver here? Probably not, so in that case we could make this a number property instead of a string, and increment on every change.